src/internal_modules/roc_audio/depacketizer.cpp

```diff
@@ -207,6 +223,9 @@ void Depacketizer::update_packet_(FrameInfo& info) {
         return;
     }
 
+    next_capture_ts_ = packet_->rtp()->capture_timestamp;
+    valid_capture_ts_ = !!next_capture_ts_;
```

Comment:
I think if valid_capture_ts_ became true, it should never become false again, otherwise it will break logic above, what do you think?

Reply:
Also: we have rtp::Validator, that terminates session if there are unexpected seqnum or timestamp jumps. I suggest to modify it to detect capture_ts jumps as well. It will handle the case when timestamp suddenly became zero after being non-zero.